hat/assets/js/apps/Iaso/domains/app/translations/fr.json

```
@@ -375,6 +375,7 @@
     "iaso.instance.last_modified_by": "Modifié par",
     "iaso.instance.last_sync_at": "Dernière synchro",
     "iaso.instance.linkOrgUnitToInstanceReferenceWarning": "Cette opération peut toujours être annulée.",
+    "iaso.instance.linkReferenceSubmissionsConfirmMessage": "Il existe {unLinkedCount} soumissions de référence non liées. Pour les lier, cochez la case correspondante.",
```

So that it matches the English text
```suggestion
    "iaso.instance.linkReferenceSubmissionsConfirmMessage": "Vous avez sélectionné {unLinkedCount} soumissions standard. Cochez cette case pour les lier à leur unité d'org. respective en tant que soumission de référence.
```

---

I don't know why the formatting is strange here :arrow_up: